src/utils/time.ts

```diff
@@ -15,7 +15,13 @@ function getContestStatuses(
       contestStatus: Status.UPCOMING,
     };
   }
-  if (isBefore(currentTime, botRaceEnd) && isAfter(currentTime, start)) {
+  if (isAfter(currentTime, end)) {
```

Comment:
Moved this up to the top, it avoids issues in local dev when the botrace time of 1 hour is longer than the contest in tests. It's also a good early exit anyway